source/_integrations/ld2410_ble.markdown

```diff
@@ -0,0 +1,25 @@
+---
+title: LD2410 BLE
+description: Instructions on how to integrate LD2410 BLE devices into Home Assistant.
+ha_category:
+  - Sensor
+  - Presence Detection
+ha_bluetooth: true
+ha_release: 2023.1
+ha_iot_class: Local Push
+ha_codeowners:
+  - '@930913'
+ha_domain: ld2410_ble
+ha_config_flow: true
+ha_platforms:
+  - sensor
+ha_integration_type: integration
+---
+
+Integrates LD2410B sensors from [Hi-Link](http://www.hlktech.net/) into Home Assistant using Bluetooth.
+
+{% include integrations/config_flow.md %}
+
+It currently provides the motion and static value. Other values, including configuring it are TODO.
+
+You can currently configure the settings for motion/static triggers by USB with either the [desktop software](https://drive.google.com/drive/folders/1p4dhbEJA3YubyIjIIC7wwVsSo8x29Fq-?usp=sharing) or the Bluetooth [mobile app](https://www.pgyer.com/Lq8p).
```

Comment:
Why is this linking to a Google drive?

Reply:
For some arcane reason, the manufacturer has seen fit to distribute the software via a google drive share. This is the link they have provided elsewhere, and I have copied it here to ease the user's journey until the configuration can be done inside Home Assistant.

Reply:
Can we point to the manufacturer's website or documentation that provides that information and links? It seems very inefficient to use the drive link on third-party websites. They can easily expire/won't update. I rather avoid them for ending up in our documentation.

Reply:
I'm not sure how. If you look at [their listing page on aliexpress](https://www.aliexpress.com/item/1005004351593073.html), [their comments on YouTube](https://www.youtube.com/watch?v=dAzHXpP3FcI&lc=UgyG9sIEjRotx485AgB4AaABAg.9h-Wb-MW1Br9itOezFW363), or their correspondence, that's the official link as far as they provide.
Short of uploading the contents into this repository and redistributing, I think we're stuck with their official link, as unofficial as it seems to us.

Reply:
Bump?

Reply:
Thanks for clarifying. Nasty, but it is what it is I guess 🤷 

../Frenck